src/qrl/core/misc/DependencyChecker.py

```diff
@@ -8,6 +8,12 @@
 
 
 class DependencyChecker:
+    """
+    Tired of dealing with subtle errors caused by version mismatches in our
+    dependencies, cyyber wrote this little class that ensures upon startup that
```

Comment:
```suggestion
    dependencies, cyyber wrote this little class that ensures upon startup that
```
I have not written this DependencyChecker, it was a contribution from Community. Better to skip the names while writing documentation.